management/server/grpcserver.go

```diff
@@ -28,7 +28,7 @@ type Server struct {
 const AllowedIPsFormat = "%s/32"
 
 // NewServer creates a new Management server
-func NewServer(config *Config, accountManager *AccountManager, peersUpdateManager *PeersUpdateManager, turnCredentialsManager TURNCredentialsManager) (*Server, error) {
+func NewServer(config *Config, accountManager *DefaultAccountManager, peersUpdateManager *PeersUpdateManager, turnCredentialsManager TURNCredentialsManager) (*Server, error) {
```

Comment:
I suggest keeping the AccountManager interface here, then we can easily replace it with some other implementation if needed.
It will also be easier to mock in tests
That is the whole purpose of interfaces :)